src/symbol/projection.js

```diff
@@ -194,8 +194,12 @@ function updateLineLabels(bucket: SymbolBucket,
         // Awkward... but we're counting on the paired "vertical" symbol coming immediately after its horizontal counterpart
         useVertical = false;
 
-        const elevation = getElevation ? getElevation({x: symbol.anchorX, y: symbol.anchorY}) : 0;
-        const anchorPos = [symbol.anchorX, symbol.anchorY, elevation, 1];
+        // Project tile anchor to globe anchor
+        const tileAnchorPoint = new Point(symbol.tileAnchorX, symbol.tileAnchorY);
+        const elevation = getElevation ? getElevation(tileAnchorPoint) : 0.0;
+        const elevatedAnchor = [ tileAnchorPoint.x, tileAnchorPoint.y, elevation];
```

Comment:
nit: Code style, extra space
```suggestion
        const elevatedAnchor = [tileAnchorPoint.x, tileAnchorPoint.y, elevation];
```